.github/workflows/build-and-release.yml

```diff
@@ -16,7 +16,14 @@ jobs:
     - name: Build sdist
       run: |
         python -m build -s
-    - name: Publish package
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      if: startsWith(github.ref, 'refs/tags') != true
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        repository_url: https://test.pypi.org/legacy/
```

Comment:
Is there a non-legacy variant of the pypi test instance?

Reply:
See https://github.com/pypa/gh-action-pypi-publish, it seems that this API is the one currently supported by the official publish action.

Reply:
Okay.